pyxem/signals/diffraction_vectors2d.py

```diff
@@ -196,34 +184,6 @@ def __ge__(self, other):
     def from_peaks(cls, **kwargs):
         raise NotImplementedError("This method is not implemented for 2D vectors")
 
-    def filter_magnitude(self, min_magnitude, max_magnitude, *args, **kwargs):
```

Comment:
Similarly, what sort of deletion is this one?

Reply:
Same as above

Reply:
Reply also as above :)